.stylelintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Rather than call this file the default `.stylelintrc`, I'd do `stylelint-config.json` so that IDEs can actually syntax highlight it.